### ISSUE TITLE
US Multiframe CIne

### ISSUE THREAD

**Issue by kofifus** [cornerstonejs/cornerstoneTools#827]
I need to implement Multiframe Ultrasound Cine in our application ...

Does cornerstone has build in tools/support for that ?

I seem to remember seeing this  but all the tools example pages are down - can this please be fixed - (we were using those pages on a very regular basis!)

Is there any existing example and/or code ?

Thanks

**Comment by swederik** [comment]
You can use playClip. It has the same API as in 2.x (as far as I know), so you can just check out an older commit locally to see the examples.

The code is here: https://github.com/cornerstonejs/cornerstoneTools/blob/master/src/stackTools/playClip.js#L96

I believe all you need is stack tool data to play through it.

**Comment by Zaid-Safadi** [comment]
@kofifus  not sure if the examples have what you need but you can find v2.4 here:
https://rawgit.com/cornerstonejs/cornerstoneTools/v2.4.x/examples/index.html

**Comment by kofifus** [comment]
@Zaid-Safadi as far as I can tell those examples don't work

**Comment by dannyrb** [comment]
Ahh. That's weird. I assumed `unpkg` (the CDN we use for examples) returned the latest _semver_ version of our published package. It looks like NPM's `@latest` tag defaults to, well, the most recent release, and that's what `unpkg` uses when determining loosely specified packages.

tl;dr - we need to update our examples to use the latest package for a specified major version

@kofifus, this is what's effecting your #773 as well

**Comment by kofifus** [comment]
I am trying to figure it out from the code as there is very little documentation and the examples don't work

What is the difference between timeSeriesPlayer and playClip ?

**Comment by swederik** [comment]
A time series is used for 4D data (multiple stacks of multiple slices), playClip is used for 3D data (multiple slices). You want playClip.

**Comment by kofifus** [comment]
Got it thx !
